scripts/codemods/ac3-to-ac4/src/renames.ts

```diff
@@ -710,6 +710,21 @@ export const renames: Array<IdentifierRename | ModuleRename> = [
       importType: "type",
     })
   ),
+  ...[
+    { from: "isNetworkRequestInFlight" },
+    { from: "isNetworkRequestSettled" },
+  ].map(
+    moveInto({
+      from: {
+        module: "@apollo/client/core/networkStatus",
+        alternativeModules: ["@apollo/client/core", "@apollo/client"],
+      },
+      to: {
+        module: "@apollo/client/utilities",
```

Comment:
```suggestion
module: "@apollo/client/utilities",
alternativeModules: [],
```
otherwise it would be moved from `@apollo/client/core/networkStatus`, but not from `@apollo/client/core` or `@apollo/client`.